source/includes/_openbanking.md

```diff
@@ -0,0 +1,374 @@
+```
+
+> Base Open Banking URL in Production
+
+```shell
+https://openbanking.transferwise.com
+```
+
+The TransferWise Open Banking API is a collection of RESTful APIs that enable Third Party Providers (TPPs) to access account information, initiate payments and confirm availability of funds on behalf of TransferWise customers. The API implementation follows the [Open Banking UK standard](https://standards.openbanking.org.uk/api-specifications/), and TransferWise is a registered member of the [Open Banking Directory](https://www.openbanking.org.uk/providers/directory/).
+
+Through this API you can either query account information, if you're an Account Information Service Provider (**AISP**), initiate payments, if you're a Payment Initiation Service Provider (**PISP**), or confirm the availability of funds, if you're a Card Based Payment Instrument Issuer (**CBPII**). We are currently supporting version [3.1.1](https://standards.openbanking.org.uk/api-specifications/read-write-specs/v3-1-1/) of the Open Banking API.
+
+## Prerequisites
+
+> Well-Known Open Banking URL
+
+```shell
+https://sandbox.transferwise.tech/openbanking/.well-known/openid-configuration
+```
+
+There's a couple of things you should be aware of in order to connect to the TransferWise Open Banking API:
+
+* Being a registered TPP under the Open Banking Directory will ease the integration process.
+* We are using [mutualTLS](https://en.wikipedia.org/wiki/Mutual_authentication) as the means for authentication. Moreover, the `CN` of your client certificate is expected to match the `clientId` under which you are registered. 
+* You need to register your client, before you'll be able to call any of the APIs. Check out the [Sandbox Access](#open-banking-api-sandbox-access) first.
+
+For a detailed description of requirements and supported algorithms please check out the Well-Known Open Banking URL.
+
+## Sandbox Access
+
+We highly recommend that you get started with connecting to our sandbox first, before moving on to production. To get started with the registration process drop us an email to `openbanking@transferwise.com`
+
+### Open Banking Directory
+
+If you're a registered TPP in the Open Banking Directory and you are using Signing and Transport certificates issue by
+Open Banking, you can easily onboard by providing either of the following things:
+
+* You can generate and send us a Software Statement Assertion (SSA) from Open Banking Directory.  
+* Alternatively, you can also just sent us your Open Banking Organization ID together with the Software Statement ID that you wish to use.
+
+In either case, please make sure to have your list of **Redirect Uri**-s correctly configured in your Software Statement, otherwise you'll not be able to go through to whole flow.
+
+### eIDAS Certificates
+
+In case you're using an eIDAS certificates instead of the ones issued by the Open Banking Directory just contact us to 
+see how we can move forward. 
```

Comment:
would it make sense to add how to contact? I assume same openbanking@transferwise.com?